src/lib.rs

```diff
@@ -262,7 +262,10 @@ mod signature;
 mod signing;
 mod verifying;
 
+#[cfg(feature = "digest")]
 pub use curve25519_dalek::digest::Digest;
+#[cfg(feature = "digest")]
+pub use sha2::Sha512;
```

Comment:
We should probably change this later to re-export this from curve like we export Digest from